### DISCUSSION TITLE
DietPi-Software | Forgejo

### DISCUSSION THREAD

**Discussion by Cs137** [MichaIng/DietPi#6133] [Software Request]
Since [gitea has been forked by codeberg](https://blog.codeberg.org/codeberg-launches-forgejo.html), it would be nice if the resulted fork, [forgejo](https://forgejo.codeberg.page/), would be supported by the dietpi software repository.

Best regards & many thanks for the work and efforts invested to maintain this nice distribution!

**Comment by MichaIng** [comment]
Background of the fork: https://forgejo.codeberg.page/@main/faq/

> ### Why fork Gitea?
> In October 2022 the domains and trademark of Gitea were transferred to a for-profit company without knowledge or approval of the community. Despite [writing an open letter](https://gitea-open-letter.coding.social/), the takeover was later confirmed. The goal of Forgejo is to continue developing the code with a healthy democratic governance.

Trying to find info from Gitea side, but the website is down: https://gitea.io/

Sounds very reasonable to add Forgejo, also since as a soft fork the Gitea implementation can be basically copied.

**Reply by Cs137** [reply]
Thanks for picking the request up! I am not aware of any statement from gitea on the topic besides [this tweet](https://twitter.com/giteaio/status/1603961675253952513), where they said the following:

> As we've mentioned elsewhere, we wish forgejo all the best and are hopeful we can continue working together.

**Reply by Joulinar** [reply]
Should that be a replacement of Gitea or an additional software title?

**Reply by MichaIng** [reply]
Additional.

**Reply by Cs137** [reply]
Are there any updates on this request?

**Reply by martin-braun** [reply]
I'd prefer Forgejo over Gitea as well.

**Comment by MichaIng** [comment]
PR up: #7071

**Comment by StephanStS** [comment]
@MichaIng : I propose to close this discussion: https://dietpi.com/docs/software/cloud/#forgejo
